tests/phpunit/admin/test-class-wc-stripe-settings-controller.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * This test makes assertions against the class WC_Stripe_Settings_Controller.
+ *
+ * @package WooCommerce_Stripe/Tests/WC_Stripe_Settings_Controller
+ */
+
+/**
+ * WC_Stripe_Settings_Controller unit tests.
+ */
+class WC_Stripe_Settings_Controller_Test extends WP_UnitTestCase {
+	/**
+	 * @var WC_Stripe_Settings_Controller
+	 */
+	private $controller;
+
+	/**
+	 * @var WC_Gateway_Stripe
+	 */
+	private $gateway;
+
+	public function setUp() {
+		parent::setUp();
+
+		require_once WC_STRIPE_PLUGIN_PATH . '/includes/admin/class-wc-stripe-settings-controller.php';
+		$this->controller = new WC_Stripe_Settings_Controller();
+		$this->gateway    = new WC_Gateway_Stripe();
+
+	}
+
+	public function tearDown() {
+		parent::tearDown();
+		delete_option( 'woocommerce_stripe_settings' );
```

Comment:
No change required here but I think this shouldn't be necessary because a fresh environment is set up for every test class anyway.